### PR TITLE
Sprint3 james test - notification after 24 hours

### PR DESCRIPTION
**Pull request by perpetual-pelican**
Incorporated Ayush's quote code and added notification to display when a user hasn't posted for 24 hours.

### CODE REVIEW

File: mhapsite/mhap/views.py [hunk 1]
```diff
@@ -205,8 +206,16 @@ def index(request):
...
     happy_graph = BarChart(data_source, options=options)
-  
+
+    # get first post (latest post) from queryset
     instance = queryset.first()
+    # if the user has a latest post, display a message if it is older than a day
+    if instance:
+        # this makes sure the dates are in the same format:
+        current_time = datetime.datetime.utcnow().replace(tzinfo=utc)
+        difference = current_time - instance.updated
```
Comment (akohli96) [inline mhapsite/mhap/views.py:216]: Looks good

**Review by akohli96 — APPROVED** [review]

File: fixtures/quotes.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    "quote": "All seasons are beautiful for the person who carries happiness within.",
+    "author": "Horace Friess"
+  }
+},
+{
+  "model": "mhap.quote",
+  "pk": 2,
+  "fields": {
+    "quote": "I begin with an idea and then it becomes something else.  ",
+    "author": "Pablo Picasso "
```
Comment (markiragoldberg) [inline fixtures/quotes.json]: Trailing space in quote and author both have to go!

**Review by markiragoldberg — APPROVED** [review]
Good.

**Review by akohli96 — APPROVED** [review]

**Comment by akohli96** [comment]
Mark, being QA manager. Please merge it in

**Pull request merged**